### ISSUE TITLE
Compile error when compiling with clang 9

### ISSUE THREAD

**Issue by ceeac** [azhirnov/FrameGraph#8]
```
clang: error: unknown argument: '-fchkp-check-incomplete-type'
```
clang version:
```
$ clang-9 --version
clang version 9.0.1-9ubuntu1 
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

When compiling with GCC,  it warns that `-fchkp-check-incomplete-type` is no longer supported.

**Comment by azhirnov** [comment]
For gcc and clang in cmake try set `FG_ENABLE_SIMPLE_COMPILER_OPTIONS` to ON.
Support for this compilers is a bit abandoned.
I will try to fix it somewhen.

**Comment by Green-Sky** [comment]
The `-fchkp-check-incomplete-type` is beeing set in https://github.com/azhirnov/FrameGraph/blob/dev/cmake/compilers.cmake#L220. And as far as I can tell, its gcc only.

**Comment by Green-Sky** [comment]
Also, gcc 9 complains: 
`g++-9: warning: switch ‘-fchkp-check-incomplete-type’ is no longer supported`

**Comment by clojur** [comment]
When rendering loop，Serious memory leak

**Comment by azhirnov** [comment]
I can't setup clang on CI, but it fixed in 2020.3
